products/servicedirectory/terraform.yaml

```diff
@@ -0,0 +1,65 @@
+# Copyright 2020 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Provider::Terraform::Config
+overrides: !ruby/object:Overrides::ResourceOverrides
+  Namespace: !ruby/object:Overrides::Terraform::ResourceOverride
+    import_format: ["projects/{{project}}/locations/{{location}}/namespaces/{{namespace_id}}"]
```

Comment:
This actually generates import docs that allow several formats: https://github.com/modular-magician/terraform-provider-google-beta/compare/auto-pr-3371-old..auto-pr-3371#diff-e6eaf936973dd0e1c35282867f595bedR113-R115

I think you'll need to either update this to just be "{{name}}" or change your custom import to accept all three formats (if you go with the latter option, see if any of the helpers in field_helper.go would help with that)

Reply:
Added extra custom_import formats and tests for them.